website/playground/Playground.js

```diff
@@ -242,6 +242,16 @@ class Playground extends React.Component {
                           checked={editorState.showSecondFormat}
                           onChange={editorState.toggleSecondFormat}
                         />
+                        <Checkbox
+                          label="show output"
+                          checked={editorState.showOutput}
+                          onChange={editorState.toggleOutput}
+                        />
+                        <Checkbox
+                          label="show input"
+                          checked={editorState.showInput}
+                          onChange={editorState.toggleInput}
+                        />
```

Comment:
Maybe let's change the order of the checkboxes to match the order of the panels?

Reply:
I was thinking that these two buttons won't be used as
much as others.

Reply:
That's true, but the order looks strange. Let's change it.

Reply:
I'm happen to be an expert to  cure OCD, want help? I can give you discount. 😄

Reply:
Can you edit it? I'm not near my laptop.